### PR TITLE
fix shadowing errors

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this opm-common=858 please

File: tests/compareUpscaling.cpp [hunk 1]
```diff
@@ -49,20 +49,20 @@ void startTest(const char* testFile, const char* compareFile,  double absoluteTo
...
 
-	absDev = std::abs(testVec[i]-compareVec[i]); 
+	absDev = std::abs(testVec[i]-compareVec[i]);
 	relDev = absDev/double(std::max(std::abs(testVec[i]),std::abs(compareVec[i])));
 	
 	if (relDev > relativeTolerance && absDev > absoluteTolerance){
-	    for(size_t i = 0; i < testVec.size(); i++){
-		std::cout << "For data value " << i << "- The relative deviation is "<< relDev << " greater than the limit "<< relativeTolerance << std::endl; 
+	    for(auto val : testVec) {
+		std::cout << "For data value " << val << "- The relative deviation is "<< relDev << " greater than the limit "<< relativeTolerance << std::endl;
```
Comment (atgeirr) [inline tests/compareUpscaling.cpp]: This changes the meaning of the output, but I think it is ok.

File: opm/elasticity/materials.hh [hunk 2]
```diff
@@ -29,9 +29,9 @@ public:
   //! \param[in] ID      External material number
   //! \param[in] Emod    Young's modulus
   //! \param[in] Poisson Poisson's ratio
-  //! \param[in] rho     Mass density
-  Isotropic(int ID, double Emod, double Poisson, double rho = 0.0)
-    : Material(ID,rho)
+  //! \param[in] rho_    Mass density
```
Comment (atgeirr) [inline opm/elasticity/materials.hh]: Using trailing underscores for non-members is a bit exotic... but you call the shots in the elasticity code!

File: tests/compareUpscaling.cpp (hunk 1 above)
Comment (akva2) [reply tests/compareUpscaling.cpp]: that is unfortunate, was not my intention to change it, but i guess it's okay.

**Pull request merged**